### PR TITLE
Accessibility additions

### PR DESCRIPTION
**Pull request by ccodling**
* Added basic aria labels to the day cells
* Added a file for accessibility tests and added a test for aria label content
* ran build

| Q               | A
| --------------- | ---
| Bug fix?        | no|yes
| New feature?    | yes
| BC breaks?      | no|yes
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

**Comment by ccodling** [comment]
This was forked off of RC1 so when I ran the build task all the RC1 changes showed up.  Most the changes I did not make

File: dist/js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -1947,37 +1948,37 @@
 							'</tr>'+
 						'</tfoot>'
 	};
-	DPGlobal.template = '<div class="datepicker">'+
+	DPGlobal.template = '<div class="datepicker" role="application" aria-label="Calendar view date picker">'+
```
Comment (ccodling) [inline dist/js/bootstrap-datepicker.js]: "role" and "label" additions for screen readers

File: js/bootstrap-datepicker.js [hunk 2]
```diff
@@ -1054,7 +1055,7 @@
 					clsName = $.unique(clsName);
 				}
 
-				html.push('<td class="'+clsName.join(' ')+'"' + (tooltip ? ' title="'+tooltip+'"' : '') + ' data-date="' + prevMonth.getTime().toString() + '">' + content + '</td>');
+                html.push('<td class="' + clsName.join(' ') + '"' + (tooltip ? ' title="' + tooltip + '"' : '') + ' data-date="' + prevMonth.getTime().toString() + '" data-title="' + accessibleTitle + '">' + content + '</td > ');
```
Comment (ccodling) [inline js/bootstrap-datepicker.js]: This builds the the text for a label to be used for a single day in the calendar. e.g. March 16 2017 Thursday -- I mostly followed the recommendations of another datepicker that is accessible

File: tests/suites/accessibility.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿module('Accessibility', {
```
Comment (ccodling) [inline tests/suites/accessibility.js]: Added this file to do unit tests for accessibility.  This has a single test for now to test the generated label for a calendar day.

**Review by ZhugeSong — APPROVED** [review]
The accessibility attributes look good to me.

File: tests/suites/component.js [hunk 4]
```diff
@@ -245,7 +245,9 @@ test('Focused ceil for decade/century/millenium views', function(){
       .datepicker({
         startView: 2,
         defaultViewDate: {
-          year: 2115
+          year: 2115,
```
Comment (ccodling) [inline tests/suites/component.js]: This unit test did not work so I fixed it.

**Pull request merged**